Finder.php

```diff
@@ -601,7 +601,17 @@ public function getIterator()
         $maxDepth    = $this->getMaxDepth();
         $splFileInfo = $this->getSplFileInfo();
 
-        foreach ($this->getPaths() as $path) {
+        $collection  = $this->getPaths();
+        $paths       = [];
+        foreach ($collection as $path) {
+            $paths = array_merge(
+                $paths,
+                glob($path, GLOB_ONLYDIR|GLOB_BRACE)
```

Comment:
I think you are right... I can't remember why it's in the `getIterator` method, hope there was one reason in my head :smile: